rules/aws_cloudtrail_rules/aws_cloudtrail_forge_web_credentials.py

```diff
@@ -0,0 +1,22 @@
+from panther_base_helpers import deep_get, aws_rule_context
```

Comment:
In the current form, this detection is going to alert on every successful AUTH_EVENTS. I believe this detection needs to be revisited to alert on these events when they're interesting in a security context. I'm not sure offhand how to identify when they're interesting vs when they're not

Reply:
See asana task: https://app.asana.com/0/0/1203074962567709/1203509321364067/f

Reply:
@hbenac10 please remove this detection from the PR for the time being.

Reply:
Removed files.